vsifile/io/http.py

```diff
@@ -28,77 +23,16 @@ def __hash__(self):
         """Object hash."""
         return hash((self.name, self.mode))
 
-    def __enter__(self):
-        """Open file and fetch header."""
-        logger.debug(f"Opening: {self.name} (mode: {self.mode})")
-        head = self.client.head(self.name)
-        assert head.status_code == 200
-        assert head.headers.get("accept-ranges") == "bytes"
```

Comment:
not sure if we can do this with obstore

Reply:
You mean checking the response headers?

Reply:
I think object_store assumes that all servers can handle range requests.

Reply:
yes

Reply:
👍 ok I'm fine with that :D